plugins/gnome-builder/vala_langserv.plugin.in

```diff
@@ -1,6 +1,6 @@
 [Plugin]
-Name=Vala
-Description=Vala language server
+Name=Vala Language Server
```

Comment:
I had this plugin just called "Vala" because I intended to convey that this is the only viable way to have Vala support in GNOME Builder. Eventually I hope to merge this plugin with upstream.

Reply:
Builder already has a Vala pack plugin, so I'm guessing it would become part of that. And since we're distributing this to users already, this PR makes it clearer which plugin does what.

Reply:
@benwaffle no, the Vala pack plugin I think is being deprecated.

Reply:
I've thought about this some more and I'm okay with it now.